roles/openshift_manage_node/tasks/config.yml

```diff
@@ -22,7 +22,7 @@
     - node_status.results.results | length > 0
     - node_status.results.results[0]['items']
         | map(attribute='metadata.annotations') | map('list') | flatten
-        | select('match', '[\"node.openshift.io/md5sum\"]') | list | length ==
+        | select('match', 'node.openshift.io/md5sum') | list | length ==
```

Comment:
It's hard to tell whether this would work or not just by looking at it.  Are you able to comment (in github, not code) with example input/output?

This kind of thing, while possible in pure ansible, might be better off as custom filter so we can do some unit testing.